client/my-sites/upgrades/checkout/credit-card-payment-box.jsx

```diff
@@ -56,14 +56,9 @@ var CreditCardPaymentBox = React.createClass( {
 	},
 
 	render: function() {
-		var classSet = classNames( {
-			'credit-card-payment-box': true,
-			selected: this.props.selected === true
-		} );
-
 		return (
 			<PaymentBox
-				classSet={ classSet }
+				classSet="credit-card-payment-box"
```

Comment:
Add this class here doesn't make sense now since we are rendering only one component. It's like the component always will be `selected`. But I understand that I have to add the animation/css when the box is rendered. Working on this. Adding other comments here as well.